exercises/G-numbers/exercise.js

```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+totalNumber = numberOfStudents + numberOfMentors;
+
+console.log(`Total numnber of students and mentors: ${totalNumber}`);
```

Comment:
I think you will need to print
"Number of students: 15
Number of mentors: 8
Total numnber of students and mentors: 23"
Currently you are only printing the last sentence